packages/twenty-front/src/modules/views/components/ViewBarFilterEffect.tsx

```diff
@@ -0,0 +1,85 @@
+import { isNonEmptyString } from '@sniptt/guards';
+import { useEffect } from 'react';
+
+import { useRecoilComponentValueV2 } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentValueV2';
+import { useSetRecoilComponentStateV2 } from '@/ui/utilities/state/component-state/hooks/useSetRecoilComponentStateV2';
+
+import { fieldMetadataItemUsedInDropdownComponentSelector } from '@/object-record/object-filter-dropdown/states/fieldMetadataItemUsedInDropdownComponentSelector';
+import { objectFilterDropdownSelectedOptionValuesComponentState } from '@/object-record/object-filter-dropdown/states/objectFilterDropdownSelectedOptionValuesComponentState';
+import { currentRecordFiltersComponentState } from '@/object-record/record-filter/states/currentRecordFiltersComponentState';
+import { CURRENT_WORKSPACE_MEMBER_SELECTABLE_ITEM_ID } from 'twenty-shared/constants';
+import { jsonRelationFilterValueSchema } from 'twenty-shared/types';
+import { isDefined } from 'twenty-shared/utils';
+import { relationFilterValueSchema } from '../view-filter-value/validation-schemas/relationFilterValueSchema';
+
+type ViewBarFilterEffectProps = {
+  filterDropdownId: string;
+};
+
+export const ViewBarFilterEffect = ({
```

Comment:
There has been a recent refactor on the filter, could you please remove this component and re-implement the logic you need ? It might be already working.